components/firebase.ts

```diff
@@ -0,0 +1,19 @@
+import { FirebaseOptions, initializeApp } from "firebase/app"
+import { getAuth } from "firebase/auth"
+
+// It's OK to check in these keys since they're bundled into the client and it
+// makes it easier to contribute. 
+//
+// TODO: Replace with production config before v2 release. Do not check in
+// production config.
+const config: FirebaseOptions =  {
```

Comment:
is it ok to commit these API keys to the repo?

Reply:
Good question, they end up included in the deployed site regardless. So even if we didn't check them into the repo, you could go to the site and find them in the javascript files. Still, in general it's best practice not to check them in, and when we launch the v2 site we'll use a different set of keys that aren't checked in.

The benefit to including the development keys is that it's much easier for people to contribute on the frontend. `yarn dev` will let them work on the frontend and interact with all the data on the backend without needing to think about keys.